src/Microsoft.Health.Fhir.SqlServer/Features/Search/SqlServerSearchService.cs

```diff
@@ -55,6 +57,9 @@ internal class SqlServerSearchService : SearchService
         private readonly BitColumn _isMatch = new BitColumn("IsMatch");
         private readonly BitColumn _isPartial = new BitColumn("IsPartial");
         private readonly SqlConnectionWrapperFactory _sqlConnectionWrapperFactory;
```

Comment:
> private readonly SqlConnectionWrapperFactory _sqlConnectionWrapperFactory; [](http://example.com/codeflow?start=7&length=75)

Can you remove this completely so we have one fully clean class?

Reply:
SqlConnectionWrapperFactory not used any more